### PR TITLE
spook 0.5.5 (new formula)

### PR DESCRIPTION
**Pull request by grimnight**

### CODE REVIEW

File: Library/Formula/spook.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  homepage "https://github.com/johnae/spook"
+  url "https://github.com/johnae/spook.git",
+      :tag => "0.5.3",
+      :revision => "029dd29bec73b9262a7760fb9c7b0555b19681d8"
+  head "https://github.com/johnae/spook.git", :branch => "master"
+
+  depends_on "cmake" => :build
+
+  def install
+    ENV.deparallelize
```
Comment (bfontaine) [inline Library/Formula/spook.rb]: Is upstream aware of that?

File: Library/Formula/spook.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    ENV["PREFIX"] = "#{prefix}"
```
Comment (bfontaine) [inline Library/Formula/spook.rb]: You should be able to use `prefix` instead of `"#{prefix}"`.

File: Library/Formula/spook.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  def caveats; <<-EOS.undent
+    It's a good idea to first run spook with the setup switch on a new system. The way you do that is to just run:
+      spook --setup
+    That creates #{ENV["HOME"]}/.spook/notifiers/default/terminal_notifier.moon (and the needed directories). It's useful to keep that
+    even when you create your own notifiers. It provides meaningful terminal output even though you might also want say OS X
+    notifications, growl notifications or some other type of notifier (like a tmux notifier or linux notifier using notify-send).
+    For your own notifiers, just create directories under $HOME/.spook/notifiers and place your notifiers there. Any notifiers
+    from any sub-directories will automatically load.
+    EOS
+  end
```
Comment (bfontaine) [inline Library/Formula/spook.rb]: Please remove anything non-Homebrew-specific here.

File: Library/Formula/spook.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Note: pull from git tag to get submodules
+class Spook < Formula
+  desc "Replacement for Guard, based on LuaJIT, MoonScript, C and libuv"
+  homepage "https://github.com/johnae/spook"
+  url "https://github.com/johnae/spook.git",
+      :tag => "0.5.5",
+      :revision => "7d1877eff3c2dd7a7f3fb5a9ad160d38ce84ba39"
+  head "https://github.com/johnae/spook.git", :branch => "master"
```
Comment (DomT4) [inline Library/Formula/spook.rb]: We assume `master` is the default so doesn't need to be written.

File: Library/Formula/spook.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+      :tag => "0.5.5",
+      :revision => "7d1877eff3c2dd7a7f3fb5a9ad160d38ce84ba39"
+  head "https://github.com/johnae/spook.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    ENV["PREFIX"] = prefix
+    ENV["CC"] = ENV.cc
+    system "make"
```
Comment (bfontaine) [inline Library/Formula/spook.rb]: You should be able to pass these to `make`: `system "make", "PREFIX=#{prefix}", "CC=#{ENV.cc}"`.

**Comment by bfontaine** [comment]
Audit is failing:
```
==> audit problems
spook:
 * GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)
```

**Comment by DomT4** [comment]
This should be hosted in a tap until it gains enough popularity to pass `brew audit --strict --online`. We're more than happy to take another look at it at that point though!

You can find documentation on hosting in your own tap [here](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md) if interested. Thank you for the submission to Homebrew :heart:

**Pull request closed without merging**